applications/structure/bar/application.h.orig

```diff
@@ -0,0 +1,813 @@
+/*  ______________________________________________________________________
+ *
```

Comment:
Do we need this and similar `.orig` files? I would remove them if not strictly necessary.